packages/lit-html/src/static.ts

```diff
@@ -23,8 +23,9 @@ import {html as coreHtml, svg as coreSvg, TemplateResult} from './lit-html.js';
  * Static values can be changed, but they will cause a complete re-render
  * since they effectively create a new template.
  */
-export const unsafeStatic = (value: string) => ({
+export const unsafeStatic = (value: string): StaticValue => ({
   ['_$litStatic$']: value,
+  r: /_/,
```

Comment:
It's clever that this is short, but using a RegExp is maybe a little obscure? Also `typeof x === {aLiteral}` is hghly optimized in JS VMs.

What about:

```ts
const staticBrand = Symbol();
export const unsafeStatic = (value: string): StaticValue => ({
  ['_$litStatic$']: value,
  r: staticBrand,
});
```

Then:
```ts
if (!(typeof (value as Partial<StaticValue>)?.r !== 'symbol')) {
```

Reply:
IE11 doesn't have native Symbol, right, so `typeof Symbol() === 'symbol'` can't work, can it? https://caniuse.com/mdn-javascript_builtins_symbol

Reply:
We already require the Symbol polyfill.

`Symbol.for()` might be good like you mentioned before too, then you can do `(value as Partial<StaticValue>)?.r === staticBrand`

Reply:
Since these are each brand new instances of RegExp, and not a shared value, I'm assuming you're just trying to guard against JSON injections. In that case, you can take advantage of `constructor: undefined` to get the same benefits without needing an object (or a `instanceof` check, which is very slow). JSON cannot inject an `undefined` value, and `constructor` on all instances, so `value.constructor === undefined` guarantees the `value` was not constructed from JSON.

Reply:
That's a really cool trick @jridgewell! Ended up going with Symbol.for('') for clarity and to avoid hitting VM edge cases.

Reply:
Oh, erp, didn't push all my changes, should be there now